src/Jab.Roslyn3/Generator/ContainerGenerator.Roslyn3.cs

```diff
@@ -1,7 +1,9 @@
 #if !ROSLYN4_0_OR_GREATER
 namespace Jab
 {
-    public partial class ContainerGenerator: ISourceGenerator
+#pragma warning disable RS1001 // We don't want this to be discovered as analyzer but it simplifies testing
```

Comment:
Do you mind removing the extra `Generator` directory? The [other part](https://github.com/pakrym/jab/blob/main/src/Jab/ContainerGenerator.cs) of this file is not nested under any directories.

Reply:
I'm not sure how to avoid conflict between ContainerGenerator.Roslyn3 and ContainerGenerator.Unity.
The way I fixed it last time is by moving the ContainerGenerator.Roslyn3 into the Jab.Roslyn3 project and adding a `<Compile Include="Generator\ContainerGenerator.Roslyn3.cs" />`
But I'm not sure I understand your setup fully, is this problematic?

Reply:
Is ContainerGenerator.Roslyn3 getting included into Jab.Unity? Stange, considering it's in a completely different directory.

Reply:
Why different? in main branch it's inside Jab directory

Reply:
Right, but in your branch you moved it into a src/Jab.Roslyn3/Generator/ContainerGenerator.Roslyn3.cs.

I think the move is the right thing, I'd just prefer the file to be at src/Jab.Roslyn3/ContainerGenerator.Roslyn3.cs (without Generator part)

Reply:
Ahh np, after I moved it it works, I'll move it out of the directory